src/models/ngo.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require('mongoose');
+
+const ngoSchema = new mongoose.Schema({
+  ngoName: {
+    type: String,
+    required: true,
+  },
+  webSite: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+  },
+  phone: {
+    type: Number,
+    required: true,
+  },
+  publishedEvents: [
+    {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'Event',
+    },
+  ],
+  password: {
+    type: String,
+    required: true,
```

Comment:
`password` shouldn't be required, because we might use social auth like google auth